libs/hwes/command.go

```diff
@@ -0,0 +1,29 @@
+package hwes
+
+import "github.com/google/uuid"
+
+// CommandBase acts as a container for the targeted AggregateID
+// A concrete command in our CQRS pattern must call NewCommandBase
```

Comment:
> must 

no, it doesn't

if we want to enforce anything, there should be an interface for it